p4-16/psa/PSA.mdk

```diff
@@ -244,6 +244,23 @@ after the Ingress control block has completed executing, based upon
 the contents of several metadata fields in the struct
 `psa_ingress_output_metadata_t`.
 
+The function `platform_port_valid()` mentioned below takes a value of
```

Comment:
Is the intent to define `platform_port_valid()` as a _must support_ extern? We should then define it as:
```
bool platform_port_valid(in PortId_t p);
```
And it seems that we should allow this extern to be instantiated and called anywhere in the pipeline: parser, ingress/egress controls, deparser.

Reply:
My intent when writing it was simply to name it and explain it one time, since it is used twice in the pseudocode.  I wasn't thinking that this would be a function that someone would want to call in their P4 programs.

If you think they would want to, we can do as you suggest.

If there isn't much reason for a P4 program author to call it (I can't think of a good reason), I can add a sentence or so mentioning this.